lib/themes/dosomething/paraneue_dosomething/includes/preprocess/preprocess_node.inc

```diff
@@ -199,70 +193,66 @@ function paraneue_dosomething_preprocess_node_campaign(&$vars) {
 /**
  * Expands $reportbacks_gallery array with themed Reportback items and data for given node preprocess $vars.
  */
-function paraneue_dosomething_preprocess_reportback_gallery(&$vars) {
-  $reportbacks_gallery = &$vars['reportbacks_gallery'];
-
-  if (!isset($reportbacks_gallery['desired_count'])) {
-    $reportbacks_gallery['desired_count'] = 6;
+function paraneue_dosomething_preprocess_reportback_gallery(&$vars, $count = 6) {
+  $gallery = [];
```

Comment:
Would this read better to initialize all as one statement, like `$filters` below?

```php
$gallery = [
  'admin_access' => user_access('view any reportback') ? 'true' : 'false',
  'desired_count' => $count;
  // and so on...
];
```